app/styles/ui/_app.scss

```diff
@@ -45,6 +45,12 @@
             cursor: default;
           }
 
+          .branches-list-item {
+            div {
+              pointer-events: none;
+            }
+          }
```

Comment:
Why did you need this?

Reply:
Otherwise.. the cursor will not show up when hovering over it. Thus, during cherry-pick when moving your mouse over the branch-list-item you will see the copy curser for a second (when not actually on the branch name or time) then it will disappear when you go over the branch-name and then reappear when you leave the branch-name. 

Also.. this prevents the tool tip that we don't want when during cherry-picking.